swagger.yaml

```diff
@@ -3235,13 +3235,19 @@ definitions:
         format: date-time
         x-go-custom-tag: gorm:"type:timestamp with time zone"
         description: Time at which the operator was last updated.
+      version:
+        type: string
+        description: Version of the operator.
 
   operator-monitor-report:
     type: object
     properties:
       name:
         type: string
         description: Unique name of the operator.
+      version:
+        type: string
+        description: Version of the operator.
```

Comment:
Why does the version need to be reported? Are we expecting to support different versions of the same OLM?
Is there any logic/verification being made on the service/controller with the version or is it only shown to the user?

Reply:
I am not aware of any greater plans, but the reason the issue is opened is because "This is required in order to generate CRs to take advantage of  features which may be available in a specific version."

Reply:
AFAIK we do not want to maintain OLM versions

Reply:
@machacekondra i really not sure how setting version in status report will help to do something with this version? 
In my opinion this feature is not relevant till we don't have some very specific usage for it but maybe i am wrong.
@avishayt what do you think?

Reply:
@tsorya This PR is not about setting any version in status field. This feature is about reporting the version of the operator(OLM, built-in) to the `assisted-service` from `assisted-installer-controller`.

Reply:
This doesn't seem to accomplish what the JIRA intended.  From what I understand the point of the JIRA was to determine what operator version **will be** installed (before starting the installation), not what version **is being** installed (during the installation).

Reply:
Ah, sorry I misunderstand the JIRA issue then. So currently we install the OLM with `installPlanApproval: "Automatic"` meaning to have the operator up to date always, so what sense does it have to specify the version, if it will be updated immediately?

Do you want to then select `Manual` `installPlanApproval`?

Reply:
It might be nice, but nobody asked for it, so better to not have that feature.  If someone needs to know the version at this point they can check on the cluster themselves.